rancid-tomatillos/src/Components/MovieCard/MovieCard.js

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
+import './MovieCard.scss';
+import images from '../../images/images'
 
 const MovieCard = ({ id, poster, avgRating }) => {
+  let icon;
+  if (avgRating < 4) {
+    icon = images.badTomatillo;
+  } else if (avgRating > 3 && avgRating < 7) {
+    icon = images.okayTomatillo;
+  } else if (avgRating > 6) {
+    icon = images.goodTomatillo;
+  }
   return (
```

Comment:
yess love this functionality as well. The progressively rancid tomatillos to indicate difference in rating is great! And I think it makes for a better UX than having to make a selection between 1-10